src/pages/VenuePage/VenuePage.tsx

```diff
@@ -204,12 +318,25 @@ export const VenuePage: React.FC = () => {
     venue.hasPaidEvents &&
     !isUserVenueOwner
   ) {
-    if (eventRequestStatus && !event) {
-      return <>This event does not exist</>;
+    if (isEventLoaded && !event) {
+      return (
+        <>
+          <Preload venue={venue} />
+          This event does not exist
+        </>
+      );
     }
```

Comment:
At this stage i'm not convinced that `Preload` is going to have any meaningful impact on things; and if we do decide to keep it, it shouldn't end up duplicated in so many places.

Reply:
same as https://github.com/sparkletown/sparkle/pull/1713#discussion_r666071927